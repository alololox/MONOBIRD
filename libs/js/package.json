{
  "name": "root",
  "private": true,
  "devDependencies": {
    "@babel/cli": "^7.7.4",
    "@babel/core": "^7.7.4",
    "@babel/preset-env": "^7.7.4",
    "@babel/preset-react": "^7.7.4",
    "babel-core": "^7.0.0-bridge.0",
    "babel-loader": "^8.0.6",
    "babel-plugin-styled-components": "^1.10.6",
    "lerna": "^3.19.0",
    "webpack": "^4.41.2"
  },
  "scripts": {
    "build": "lerna exec --parallel -- babel src -d lib --ignore **/*.story.js,**/*.spec.js"
  },
  "dependencies": {
    "react": "^16.12.0",
    "react-dom": "^16.12.0",
    "styled-components": "^4.4.1"
  }
}
