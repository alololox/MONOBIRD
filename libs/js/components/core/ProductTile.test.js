import React from "react";
import { render } from "@testing-library/react";
import { ProductTile } from "./index.js";

describe("Dirty Little ProductTile Test", () => {
  it("Renders ProductTile", () => {
    const product = {
      name: "Alpha SV Jacket Men's",
      url: "/shop/mens/alpha-sv-jacket",
      heroImage:
        "https://images.arcteryx.com/F19/350w/Alpha-SV-Jacket-Labyrinth.jpg"
    };
    const { asFragment } = render(<ProductTile product={product} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
