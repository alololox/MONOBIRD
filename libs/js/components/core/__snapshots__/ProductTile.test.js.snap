// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dirty Little ProductTile Test Renders ProductTile 1`] = `
<DocumentFragment>
  <a
    class="sc-EHOje boeMmq"
    href="/shop/mens/alpha-sv-jacket"
  >
    <h3>
      Alpha SV Jacket Men's
    </h3>
    <img
      src="//placehold.it/250x250"
    />
  </a>
</DocumentFragment>
`;
